### DISCUSSION TITLE
Host OS requirements for v2.0?

### DISCUSSION THREAD

**Discussion by nekoniaow** [dirkwhoffmann/vAmiga#694] [Q&A]
Hi,
I downloaded v2.0 today and noticed the app would not run on my Mojave Mac since it's set with OS 10.15 as the minimum version.

Is this a definitive requirement or will there be some leeway for future releases?
I tend to follow the OS update cycle with a very large delay (as you can see) so I would definitely appreciate if the app could support older OS versions. Are there any particular OS 10.15 features that it absolutely needs to run?

Also, it may be helpful to users to have this requirements made explicit on the website, I searched for it there but could not find it anywhere and had to look into the package's plist file to find it. 😉

Thanks!

**Comment by dirkwhoffmann** [comment]
> Is this a definitive requirement or will there be some leeway for future releases?

I had to push the minimum OS version to 10.15, because the latest versions of vAmiga and VirtualC64 utilise the C++ FileSystem library. This library isn't available prior to 10.15.
